scripts/wikipedia.js

```diff
@@ -100,7 +124,18 @@ function createTooltipWindow (metadata, id) {
   )
   return $('<a>').append(text_elements, details).addClass('popup_box').attr('href', 'https://archive.org/details/' + id)[0].outerHTML
 }
-
+function createDonateAnchor(isbn){
+  let img = $('<img>')
+    .attr({ 'alt': 'Read', 'src': chrome.extension.getURL('images/icon.png') })[0]
+  let a = $('<a>')
+    .attr({
+      'href': 'https://archive.org/donate',
+      'class': 'btn-archive',
+      'style': 'padding: 5px;'
+    })
+    .prepend(img)
+  return a
+}
```

Comment:
You can write a lot more compact code, please try things like:
```
return $('<a>')
    .attr({'href': 'https://archive.org/donate',  'class': 'btn-archive', 'style': 'padding: 5px;'})
    .prepend(
      $('<img>').attr({ 'alt': 'Read', 'src': chrome.extension.getURL('images/icon.png') })[0]
    )
```
There is no point in making local variables like `img` and `a`.

Also, try using a consistent style, e.g. you put all attrs in one line for the `<img>` but then you put each attr in a different line for the `<a>`.